tests/utf8_support_test.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+import sys
+
+import pytest
+
+import depthai as dai
+
+def test_utf8_path_asset_manager(tmp_path):
+
+    FILE_CONTENTS="Hello World 👍"
+
+    tmp_sneaky_file_dir = tmp_path / "UTF-8 dir 🙈🙉🙉"
+    tmp_sneaky_file_dir.mkdir()
+    tmp_sneaky_file_path = tmp_sneaky_file_dir / "sneaky 👍 čćšđžĆČLpđšlčk file.txt"
+    tmp_sneaky_file_path.write_bytes(FILE_CONTENTS.encode("utf-8"))
```

Comment:
looks like test is failing since https://github.com/luxonis/depthai-python/commit/4240db973c0501727eaa2698c0363076005fedfb